progress/README.md

```diff
@@ -0,0 +1,44 @@
+# Firestore Standup collection data model
+
+```json
+{
+  "id": "String",          // Unique identifier for the document
+  "type": "String",        // Type of the document, e.g., "task progress" or "user progress"
+  "userId": "String",      // User ID (applicable for both task progress and user progress)
+  "taskId": "String",      // Task ID (applicable for task only)
+  "completed": "String",   // Completed portion of the task (applicable for both similar to yesterday or past progress for user)
```

Comment:
can this key be progress, completed seems to be task specific

Reply:
to maintain consistency we have named `completed` and `planned` for both task and user.